stack/CO_SYNC.c

```diff
@@ -340,6 +352,10 @@ uint8_t CO_SYNC_process(
             }
             else{
                 SYNC->curentSyncTimeIsInsideWindow = true;
+                /* Immediately continue while inside the window */
+                if(timerNext_us != NULL){
+                    *timerNext_us = 0;
+                }
```

Comment:
No need for this. This will just trigger cycles very rapidly. If sync was received or transmitted, then `ret = 1;` and all synchronous TPDOs will be processed immediately.